packages/mds-geography-author/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Looks like these `LICENSE` files are only in a few packages. Not sure why they are not either always necessary or not included in all. @karcass

Reply:
Honestly I'm not sure whether we need them in some/all/just-the-root.  I'll ask @sarob.

Reply:
After talking to @karcass , it sounds like we can punt on the license issue for now.